framework/src/modules/http_api/helpers/utils.js

```diff
@@ -137,22 +137,27 @@ const consolidatePeers = (connectedPeers = [], disconnectedPeers = []) => {
 			ipAddress,
 			options,
 			minVersion,
-			nethash,
+			networkId,
 			nonce,
 			blockVersion,
 			maxHeightPrevoted,
 			lastBlockId,
 			...restOfPeerObject
 		} = peer;
 
-		return { ip: ipAddress, ...restOfPeerObject, state: PEER_STATE_CONNECTED };
+		return {
+			ip: ipAddress,
+			networkId,
```

Comment:
what happens if the `restOfPeerObject` has `networkId` it would override know?

Reply:
I will check what's inside `peer` but if what you comment it's the case then it was also wrong before as I only changed the variable being used.

Reply:
By above code, it simply extracts out `networkId` from the peer object if it exists so there is no chance that `restOfPeerObject` will have networkId again. only `ipAddress` is changed to `ip` in the above code.

Reply:
Resolving as per @ishantiw notes. Ping @ManuGowda